src/main/steps/index.ts

```diff
@@ -60,10 +60,12 @@ const getNextIncompleteStep = (
   removeExcluded = false,
   checkedSteps: Step[] = []
 ): string => {
-  const stepForm = stepForms[step.url];
+  const stepField = stepFields[step.url];
   // if this step has a form
-  if (stepForm !== undefined) {
+  if (stepField !== undefined) {
```

Comment:
does this have to be explicit, can it be: if (stepField)

Reply:
hmmm, im not sure, it could have been added because empty objects passed but i doubt it. ill have a look.